js/tab_panel.ts

```diff
@@ -0,0 +1,183 @@
+import { html, css, nothing, LitElement, PropertyValues } from "lit";
+import { property, queryAll, queryAssignedElements } from "lit/decorators.js";
+import { legacyStyles } from "./ipywidgets_styles";
+import { classMap } from 'lit/directives/class-map.js';
+import { materialStyles } from "./styles";
```

Comment:
Mixed single and double quotes here.

Consider also leaving a space between NPM package imports and local imports.

Reply:
Ah, thank you! Done.